model/model_training/custom_datasets/extra_rm_datasets.py

```diff
@@ -0,0 +1,26 @@
+from model_training.custom_datasets.rank_datasets import HFDataset
+from reward.instructor.rank_datasets import AnthropicRLHF
+
+
+def load_anthroopic_rlhf():
+    train = AnthropicRLHF(split="train", mode="rm")
+    validation = AnthropicRLHF(split="test", mode="rm")
+
+    print(f"AnthropicRLHF extra dataset size: {len(train)}, {len(validation)}")
+
+    return train, validation
+
+
+def load_open_ai_summarize_from_feedback():
```

Comment:
Maybe it's better to just make the split manually here and in load_open_ai_summarize_from_feedback ? 
I'm not sure what's best, but the default split seems to be not balanced in terms of sizes...